Fails to build with winapi errors

```
   Compiling spotify v0.8.0
error[E0432]: unresolved import `winapi::minwindef`
 --> C:\Users\ShayBox\.cargo\registry\src\github.com-1ecc6299db9ec823\spotify-0.8.0\src\windows_process.rs:7:13
  |
7 | use winapi::minwindef::{DWORD, FALSE, TRUE};
  |             ^^^^^^^^^ could not find `minwindef` in `winapi`

error[E0432]: unresolved import `winapi::tlhelp32`
 --> C:\Users\ShayBox\.cargo\registry\src\github.com-1ecc6299db9ec823\spotify-0.8.0\src\windows_process.rs:8:13
  |
8 | use winapi::tlhelp32::{PROCESSENTRY32, TH32CS_SNAPPROCESS};
  |             ^^^^^^^^ could not find `tlhelp32` in `winapi`

error[E0432]: unresolved import `winapi::winnt`
 --> C:\Users\ShayBox\.cargo\registry\src\github.com-1ecc6299db9ec823\spotify-0.8.0\src\windows_process.rs:9:13
  |
9 | use winapi::winnt::{HANDLE, PROCESS_ALL_ACCESS};
  |             ^^^^^ could not find `winnt` in `winapi`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `spotify` due to 3 previous errors
```

It appears the imports are incorrect, I don't know if you updated winapi without building, but they're
```
use winapi::shared::minwindef::{DWORD, FALSE, TRUE};
use winapi::um::tlhelp32::{PROCESSENTRY32, TH32CS_SNAPPROCESS};
use winapi::um::winnt::{HANDLE, PROCESS_ALL_ACCESS};
```
You also need the `tlhelp32` feature of `winapi`  
But there's another error I don't quite understand
```
error[E0308]: mismatched types
    --> src\windows_process.rs:46:46
     |
46   |         if unsafe { Process32First(snapshot, &mut entry) == TRUE } {
     |                     --------------           ^^^^^^^^^^ expected struct `winapi::tlhelp32::PROCESSENTRY32`, found struct `PROCESSENTRY32`
     |                     |
     |                     arguments to this function are incorrect
     |
     = note:    expected raw pointer `*mut winapi::tlhelp32::PROCESSENTRY32`
             found mutable reference `&mut PROCESSENTRY32`
note: function defined here
    --> C:\Users\ShayBox\.cargo\registry\src\github.com-1ecc6299db9ec823\kernel32-sys-0.2.2\src\lib.rs:1903:12
     |
1903 |     pub fn Process32First(hSnapshot: HANDLE, lppe: LPPROCESSENTRY32) -> BOOL;
     |            ^^^^^^^^^^^^^^

error[E0308]: mismatched types
    --> src\windows_process.rs:49:50
     |
49   |                 unsafe { Process32Next(snapshot, &mut entry) == TRUE }
     |                          -------------           ^^^^^^^^^^ expected struct `winapi::tlhelp32::PROCESSENTRY32`, found struct `PROCESSENTRY32`
     |                          |
     |                          arguments to this function are incorrect
     |
     = note:    expected raw pointer `*mut winapi::tlhelp32::PROCESSENTRY32`
             found mutable reference `&mut PROCESSENTRY32`
note: function defined here
    --> C:\Users\ShayBox\.cargo\registry\src\github.com-1ecc6299db9ec823\kernel32-sys-0.2.2\src\lib.rs:1905:12
     |
1905 |     pub fn Process32Next(hSnapshot: HANDLE, lppe: LPPROCESSENTRY32) -> BOOL;
     |            ^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0308`.
error: could not compile `spotify` due to 2 previous errors
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -20,11 +20,11 @@ name = "spoticli"
 path = "src/bin/spoticli.rs"
 
 [dependencies]
-winapi = "0.3.9"
+json = "0.12.4"
 kernel32-sys = "0.2.2"
 reqwest = "0.9"
-json = "0.12.4"
 time = "0.1"
+winapi = { version = "0.3.9", features = ["tlhelp32"] }
 
 [build-dependencies]
 skeptic = "0.13"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -21,7 +21,6 @@ path = "src/bin/spoticli.rs"
 
 [dependencies]
 json = "0.12.4"
-kernel32-sys = "0.2.2"
 reqwest = "0.9"
 time = "0.1"
 winapi = { version = "0.3.9", features = ["tlhelp32"] }

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -6,7 +6,6 @@
 
 // Extern crates
 extern crate json;
-extern crate kernel32;
 extern crate reqwest;
 extern crate time;
 extern crate winapi;

--- a/src/windows_process.rs
+++ b/src/windows_process.rs
@@ -1,11 +1,13 @@
 #![cfg(windows)]
 
-use kernel32::{CreateToolhelp32Snapshot, OpenProcess, Process32First, Process32Next};
 use std::cmp::Ordering;
 use std::ffi::{CStr, CString};
 use std::mem::{size_of, zeroed};
 use winapi::shared::minwindef::{DWORD, FALSE, TRUE};
-use winapi::um::tlhelp32::{PROCESSENTRY32, TH32CS_SNAPPROCESS};
+use winapi::um::processthreadsapi::OpenProcess;
+use winapi::um::tlhelp32::{
+    CreateToolhelp32Snapshot, Process32First, Process32Next, PROCESSENTRY32, TH32CS_SNAPPROCESS,
+};
 use winapi::um::winnt::{HANDLE, PROCESS_ALL_ACCESS};
 
 /// The `WindowsProcess` struct.

--- a/src/windows_process.rs
+++ b/src/windows_process.rs
@@ -4,9 +4,9 @@ use kernel32::{CreateToolhelp32Snapshot, OpenProcess, Process32First, Process32N
 use std::cmp::Ordering;
 use std::ffi::{CStr, CString};
 use std::mem::{size_of, zeroed};
-use winapi::minwindef::{DWORD, FALSE, TRUE};
-use winapi::tlhelp32::{PROCESSENTRY32, TH32CS_SNAPPROCESS};
-use winapi::winnt::{HANDLE, PROCESS_ALL_ACCESS};
+use winapi::shared::minwindef::{DWORD, FALSE, TRUE};
+use winapi::um::tlhelp32::{PROCESSENTRY32, TH32CS_SNAPPROCESS};
+use winapi::um::winnt::{HANDLE, PROCESS_ALL_ACCESS};
 
 /// The `WindowsProcess` struct.
 #[derive(Clone)]

